### PR TITLE
Add AdGuard Home docs

### PR DESCRIPTION
**Pull request by Joulinar**
DietPi-Software | AdGuard Home - add new software title

### CODE REVIEW

File: docs/software/dns_servers.md [hunk 1]
```diff
@@ -215,4 +216,82 @@ DietPi Blog: [`Pi-Hole & Unbound: How to have ad-free & safer internet in just f
...
+    - If you are running in DHCP mode, select *Change Mode*, then select: *Copy Current address to Static*
+    - Select *Static DNS* from the list, then choose a DNS server, or manually enter a custom entry.
+    - Once completed, select *Apply* to save the changes.
+
+=== "Updating AdGuard Home"
+
+    To update AdGuard Home, simply reinstall it:
+
+    ```sh
+    dietpi-software reinstall 126
```
Comment (MichaIng) [inline docs/software/dns_servers.md:270]: But it has an internal updater as well, which should be preferred as long as web UI (didn't check CLI) works.

Comment (Joulinar) [reply docs/software/dns_servers.md:270]: Nope it will just show the new version and link you to GitHub. At least this was the behaviour on my test.

Comment (MichaIng) [reply docs/software/dns_servers.md:270]: Ah, good to know. I found a post in their forums which seems to imply that an updater exist, but probably I misinterpreted it or it did but does not anymore.

Comment (Joulinar) [reply docs/software/dns_servers.md:270]: I will run a rest again

**Review by MichaIng — APPROVED** [review]

**Comment by MichaIng** [comment]
I'll merge it, so we can prepare a dev => master PR.

**Comment by Joulinar** [comment]
I need to revert my statement a little bit. It seems there is an automatic update function. However it's not working with our implementation as it seems the updater expect the executable on same directory as the config file

```
Jun 22 13:31:37 DietPi4 AdGuardHome[2973]: 2021/06/22 13:31:37 [info] Updating from v0.105.2 to v0.106.3.  URL:https://static.adguard.com/adguardhome/release/AdGuardHome_linux_arm64.tar.gz
Jun 22 13:31:37 DietPi4 AdGuardHome[2973]: 2021/06/22 13:31:37 [info] executable file /mnt/dietpi_userdata/adguardhome/AdGuardHome doesn't exist
```

🙄

**Comment by MichaIng** [comment]
Oh, that is bad. I guess it expects it to be located in the working directory then, as this is all that we change. Strange that the related CLI option exists, when it expectedly breaks the updater 🤔. Probably something we should report to AdGuard guys, to derive the install dir form the location of the currently running executable, rather than assuming that it's located in the working directory.

**Comment by Joulinar** [comment]
basically we would have 3 option

1. keep everything in `/opt/adguardhome`
2. move everything to `/mnt/dietpi_userdata/adguardhome`
3. leave as is and hope behaviour will change in future 

Currently AGH devs are working on v0.107 and it might be released short after our release. Means, we will have user reporting issues using the internal update function right after. For the time being, we could go with option 1, as this is similar to using AGH installation method and in parallel check with AGH developer if this intended or if they plan to split working dir from running executable. I mean you are right, using `-w` option is a little but useless at the moment.

**Comment by MichaIng** [comment]
FYI: https://github.com/AdguardTeam/AdGuardHome/issues/3286

**Pull request merged**